src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.NetCoreApp.cs

```diff
@@ -5,35 +5,6 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
```

Comment:
Does it require passing the [API review process](http://aka.ms/api-review)?

Reply:
Well, that's legacy link came from dotnet/corefx, not applicable to this library anymore.

Reply:
So, It's good if we remove it.